lib/models/device.coffee

```diff
@@ -1345,6 +1345,45 @@ getDeviceModel = (deps, opts) ->
 						force: Boolean(options.force)
 		.asCallback(callback)
 
+	###*
+	# @summary Get the supervisor state on a device
+	# @name getSupervisorState
+	# @public
+	# @function
+	# @memberof balena.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# balena.models.device.getSupervisorState('7cf02a6');
```

Comment:
Please also add a `.then()` like we have in the rest of the methods.
```suggestion
	# balena.models.device.getSupervisorState('7cf02a6').then(function(state) {
	# 	console.log(state);
	# });
```

Reply:
I used the `update` method as a template, which has no `.then()`, but happy to add it if you prefer.

Reply:
Addressed in latest push